index.bs

```diff
@@ -174,9 +174,8 @@ When an item is of type <dfn export for="AV1 Image Item Type">av01</dfn>, it is
     - The content of an [=AV1 Image Item=] is called the <dfn export>AV1 Image Item Data</dfn> and shall obey the following constraints:
         - <assert>The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].</assert>
         - <assert>The [=AV1 Image Item Data=] shall have exactly one [=Sequence Header OBU=].</assert>
-        - If the [=AV1 Image Item Data=] consists of a single frame (i.e. when using a single layer),
-            - <assert>It should have its <code>[=still_picture=]</code> flag set to 1.</assert>
-            - <assert>It should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.</assert>
+
+NOTE: File writers may want to set the <code>[=still_picture=]</code> and <code>[=reduced_still_picture_header=]</code> flags to 1 when possible in the [=Sequence Header OBU=] part of the [=AV1 Image Item Data=] so that AV1 header overhead is minimized.
```

Comment:
Nit/optional: "may want to" sounds rather weak. I think "should" sounds better here. But I guess "should" would make this a recommendation, which is what we wanted to avoid in the first place?